src/datagrid/components/Table/GroupFooterRenderer.tsx

```diff
@@ -0,0 +1,35 @@
+import { memo } from 'react';
+// import type { RenderCellProps } from '../../../components';
+import type { DataTableRowProps, TDataTableRow } from 'src/components/DataTable/types';
```

Comment:
We should use alias here. '@bambooapp/bamboo-molecules' or '@bambooapp/bamboo-molecules/components'